packages/roosterjs-editor-core/lib/corePlugins/CopyPastePlugin.ts

```diff
@@ -60,7 +60,7 @@ export default class CopyPastePlugin implements PluginWithState<CopyPastePluginS
     initialize(editor: IEditor) {
         this.editor = editor;
         this.disposer = this.editor.addDomEventHandler({
-            paste: this.onPaste,
+            paste: e => this.onPaste(this.editor!, e),
```

Comment:
hmm, although this.editor is not likely to be null when this function is called, but better still have a check. And you can do it inside this.onPaste